sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -72,6 +72,21 @@ def process(self, prediction_result):
     yield prediction_result.inference
 
 
+class FakeInferenceRunnerNeedsBigBatch(FakeInferenceRunner):
+  def run_inference(self, batch, unused_model):
+    if len(batch) < 100:
+      raise ValueError('Unexpectedly small batch')
+    return batch
+
+
+class FakeLoaderWithBatchArgForwarding(FakeModelLoader):
+  def get_inference_runner(self):
+    return FakeInferenceRunnerNeedsBigBatch()
+
+  def batch_elements_kwargs(self):
+    return {'min_batch_size': 9999}
```

Comment:
If the goal is to be able to set a max batch size of 1, I think it might also be useful to explicitly test the case in which `batch_elements_kwargs` returns `{max_batch_size=1}`.

Reply:
It makes the test much more difficult to write, since the actual batch size chosen in that case would be nondeterministic, so it's difficult to distinguish between size 1 batches due to randomness and size 1 batches due to forwarding the correct args.